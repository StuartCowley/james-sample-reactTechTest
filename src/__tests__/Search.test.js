import React from "react";
import { render } from "@testing-library/react";
import SearchComponent from "../components/Search";

describe("it renders correcty", () => {
  const { asFragment } = render(<SearchComponent />);
  it("renders a search box", () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
