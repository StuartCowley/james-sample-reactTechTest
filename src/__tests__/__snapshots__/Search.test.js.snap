// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correcty renders a search box 1`] = `
<DocumentFragment>
  <input
    class="search-box"
    type="text"
  />
</DocumentFragment>
`;
